deliver/lib/deliver/upload_metadata.rb

```diff
@@ -451,6 +459,42 @@ def retry_if_nil(message, tries: 5, wait_time: 10)
       end
     end
 
+    # Checking if the metadata to update includes localised App Info
+    def updating_localised_app_info?(options, app)
+      app_info = fetch_edit_app_info(app) || fetch_live_app_info(app)
+      unless app_info
+        UI.important('Can\'t find edit or live App info. Skipping upload.')
+        return false
+      end
+      localizations = app_info.get_app_info_localizations
+
+      LOCALISED_APP_VALUES.keys.each do |key|
+        current = options[key]
+        next unless current
+
+        unless current.kind_of?(Hash)
+          UI.error("Error with provided '#{key}'. Must be a hash, the key being the language.")
+          next
+        end
+
+        current.each do |language, value|
+          next unless value.to_s.length > 0
```

Comment:
I think this line is redundant (given the next 2 LOC):

```suggestion
```

Unless I'm missing something

Reply:
Good call!